.github/workflows/pr.yml

```diff
@@ -17,6 +17,12 @@ jobs:
       REACT_APP_ETHEREUM_NODE_URL: https://mainnet.infura.io/v3/d734c7eebcdf400185d7eb67322a7e57
       REACT_APP_FEATURE_COSMOS_INVESTOR: true
       REACT_APP_FEATURE_PLUGIN_COSMOS: true
+      REACT_APP_GEM_COINIFY_SUPPORTED_COINS: https://api.gem.co/institutions/coinify/supported_currencies
+      REACT_APP_GEM_WYRE_SUPPORTED_COINS: https://api.gem.co/institutions/wyre/supported_currencies
+      REACT_APP_GEM_ASSET_LOGO: https://gem-widgets-assets.s3-us-west-2.amazonaws.com/currencies/crypto/
+      REACT_APP_GEM_ENV: production
+      REACT_APP_GEM_API_KEY: 17baca45f492df6b8974d4f832a659c92644acafea3284cfb0d52550dd3bde1a
+      REACT_APP_GEM_URL: https://onramp.gem.co
```

Comment:
How do we feel about these? IMO, since this is a public, external endpoint, we do not need to "hide" it into ENV

Reply:
this is just an id not a secret, nothing is hidden